src/components/List/components/ListItem.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {block} from '../../utils/cn';
+import {DragHandleIcon} from './DragHandleIcon';
+import type {ListProps, ListItemData, ListSortHandleAlign} from '../types';
+
+const b = block('list');
+
+type ListItemProps<T> = {
```

Comment:
Maybe add default type like in other interfaces?

Reply:
Why? We don`t export this type. Type `T` come from `ListItem`

Reply:
ok